docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md

```diff
@@ -92,40 +109,43 @@ The Sumo Logic integration requires the following:
 
 To create an export configuration, do the following:
 
-1. On the **Integrations** page, click **Configure** for the Sumo Logic provider or, if a configuration is already available, **Add Configuration**.
+1. On the **Integrations** page, click **Configure** for the **Sumo Logic** integration or, if a configuration is already available, **Add Configuration**.
 1. Enter a name for the configuration.
 1. Enter your installation token, access ID, and access key.
 1. Optionally, click **Download** to download the Sumo Logic dashboard template. After you install the [YugabyteDB app](https://help.sumologic.com/docs/get-started/apps-integrations/) (coming soon) in your Sumo Logic account, you can import this JSON format template and use it as a starting point for visualizing your cluster data.
 1. Click **Test Configuration** to make sure your connection is working.
 1. Click **Create Configuration**.
 
-  {{% /tab %}}
-
-  {{% tab header="Prometheus" lang="prometheus" %}}
+### Prometheus
 
-Prometheus integration is {{<tags/feature/tp>}} and only available for clusters deployed on AWS.
+Prometheus integration is only available for clusters deployed on AWS or GCP.
 
 The Prometheus integration requires the following:
 
 - Prometheus instance
-  - deployed in a VPC on AWS
-  - [OLTP Receiver](https://prometheus.io/docs/prometheus/latest/querying/api/#otlp-receiver) feature flag enabled
-  - publically-accessible endpoint URL that resolves to the private IP of the Prometheus instance; the DNS for the endpoint must be in a public hosted zone in AWS.
+  - Deployed in a VPC on AWS or GCP.
+  - [OLTP Receiver](https://prometheus.io/docs/prometheus/latest/querying/api/#otlp-receiver) feature flag enabled.
+  - Publicly-accessible endpoint URL that resolves to the private IP of the Prometheus instance.
+
+    The DNS for the endpoint must be in a publicly accessible DNS record, allowing it to resolve globally. This typically involves adding the URL to a public DNS zone. For example, in AWS, this would mean adding the URL to a Public Hosted Zone in Route 53. To confirm that the address is publicly resolvable, you can use a tool like nslookup.
+
   - VPC hosting the Prometheus instance has the following Inbound Security Group rules:
-    - Allow HTTP inbound traffic on port 80 for Prometheus endpoint URL (HTTP)
-    - Allow HTTPS inbound traffic on port 443 for Prometheus endpoint URL (HTTPS)
+    - Allow HTTP inbound traffic on port 80 for Prometheus endpoint URL (HTTP).
+    - Allow HTTPS inbound traffic on port 443 for Prometheus endpoint URL (HTTPS).
 
     See [Control traffic to your AWS resources using security groups](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-security-groups.html) in the AWS documentation.
```

Comment:
Need something similar here for GCP, perhaps linking here https://cloud.google.com/firewall/docs/firewalls. cc: @atharva-yb